src/index.ts

```diff
@@ -0,0 +1,175 @@
+// Native
+import { Server, IncomingMessage, ServerResponse } from "http";
+import { Stream } from "stream";
+
+// Packages
+import contentType from "content-type";
+import getRawBody from "raw-body";
+import { readable } from "is-stream";
+
+import { HttpError } from "./error";
+
+const { NODE_ENV } = process.env;
+const DEV = NODE_ENV === "development";
+
+export type ResponseObject = null | Buffer | Stream | object | number | string;
+
+export type RequestHandler = (
+	req: IncomingMessage,
+	res: ServerResponse
+) => ResponseObject | undefined;
+
+export const serve = (fn: RequestHandler) =>
+	new Server((req, res) => run(req, res, fn)); // TODO: We should create an issue in DefinitelyTypes: `Server` can be called without new
+
+export default serve;
```

Comment:
This is a breaking change, as `const micro = require('micro')` will hold an object that has `.default`. I'd change it back to `module.exports = serve`

Reply:
I fixed it. Tests are now exactly the same as before.